doc/classes/CPUParticles3D.xml

```diff
@@ -109,10 +109,10 @@
 			Each particle's rotation will be animated along this [Curve]. Should be a unit [Curve].
 		</member>
 		<member name="angle_max" type="float" setter="set_param_max" getter="get_param_max" default="0.0">
-			Maximum angle.
+			Maximum initial rotation applied to each particle, in degrees.
 		</member>
 		<member name="angle_min" type="float" setter="set_param_min" getter="get_param_min" default="0.0">
-			Minimum angle.
+			Minimum equivalent of [member angle_max].
```

Comment:
In CPUParticles2D, you are replacing "equivalent of" with the full text, but then are introducing that wording here. That's really strange. I think we should just go all in and remove the "equivalent of" text.

```suggestion
			Minimum initial rotation applied to each particle, in [i]degrees[/i].
```

Reply:
There's actually many, many other "minimum equivalent of" in **ParticleProcessMaterial** that, ideally, would also need to change as a result.